cli/cmd/policy_update.go

```diff
@@ -95,12 +117,25 @@ func updateQueryFromLibrary(id string) error {
 
 func updatePolicy(cmd *cobra.Command, args []string) error {
 	var (
-		msg          string = "unable to update policy"
+		msg          = "unable to update policy"
 		err          error
 		queryUpdated bool
 		policyID     string
 	)
 
+	// if severity flag is provided, attempt bulk update
+	if policyCmdState.Severity != "" {
+		if !lwseverity.IsValid(policyCmdState.Severity) {
+			return errors.New(fmt.Sprintf("invalid severity %q' valid severities are: %s", policyCmdState.Severity, lwseverity.ValidSeverities.String()))
+		}
+
+		err = policyBulkUpdate(args)
+		if err != nil {
+			return err
```

Comment:
use `msg` here to wrap this error message? 🤔

Reply:
Going to leave this as is, as another code review comment suggests wrapping the err further down in the policyBulkUpdate func